src/accelerate/checkpointing.py

```diff
@@ -126,12 +126,13 @@ def load_accelerator_state(input_dir, models, optimizers, schedulers, process_in
             The current process index in the Accelerator state
         scaler (`torch.cuda.amp.GradScaler`, *optional*):
             An optional *GradScaler* instance to load
+        model_load_kwargs (`dict`): Additional arguments to pass to the model's `load_state_dict` method.
```

Comment:
Also bring the earlier suggestions into this section (including the spacing Sylvain suggested earlier)